lms/templates/certificates/invalid.html

```diff
@@ -6,41 +6,44 @@
 
```

Comment:
NOTE: still need to confirm that this view is needed and if so, styling/rendering is still incomplete.